benchmark/sql_regressions/DoltReleaseBenchmarkJenkinsfile

```diff
@@ -0,0 +1,48 @@
+pipeline {
+    agent none
+    stages {
+        stage('Test Dolt release') {
+            parallel {
+                stage ("Update Liquidata/dolt-sql-performance:releases") {
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
+                        TMPDIR = "${pwd()}/tempDir"
+                        DOLT_ROOT_PATH="${pwd()}/tempRoot"
+                        DOLT_CREDS = credentials("system-account-dolthub-creds")
+                        DOLT_GLOBAL_CONFIG = credentials("system-account-dolthub-config")
+                    }
+                    steps {
+                        sh "rm -rf $TMPDIR && mkdir $TMPDIR"
+                        sh "rm -rf $DOLT_ROOT_PATH && mkdir $DOLT_ROOT_PATH"
+                        dir ("dolt-release") {
+                           checkout([$class: 'GitSCM', branches: [[name: "${DOLT_RELEASE}"]],
+                                 userRemoteConfigs: [[url: 'https://github.com/liquidata-inc/dolt.git']]])
+                        }
+                        dir ("dolt-release/go") {
+                            sh "go get -mod=readonly ./..."
+                            sh "go build -mod=readonly -o ../../.ci_bin/dolt ./cmd/dolt/."
```

Comment:
Tried to install the binary from the releases page but kept getting permission denied even after `chmod a+x` and `chmod +x`... can keep iterating on this, but not sure if I'll need to be `root` to execute the binary.